doc/md_src_files/082_relatedSystems.md

```diff
@@ -105,3 +105,74 @@ In that sense, it is compatible with any of methods mentioned above or SSVC.
 However, SSVC would be better suited to address vPrioritizer's broad spectrum asset management data.
 For example, vPrioritizer aims to collect data points on topics such as asset significance.
 Asset significance could be expressed through the SSVC decision points of  [*Mission Impact*](#mission-impact) and situated [*Well-being Impact*](#well-being-impact), but it does not have a ready expression in CVSS, EPSS, or VPR.
+
+
+## SSVC using Current Information Sources
+
+Some SSVC decision points can be informed or answered by currently available information feeds or sources.
+These include two aspects of [*Exploitation*](#exploitation), [*System Exposure*](#system-exposure), [*Technical Impact*](#technical-impact), and [*Public Safety Impact*](#public-safety-impact).
+This section provides an overview of some options; we cannot claim it is exhaustive.
+Each decision point has a subsection for `Gathering Information About` it.
+These sections provide suggestions that would also contribute to creating or honing information feeds. 
+However, if there is a viable information source we have not captured, please create an [issue on the SSVC GitHub page](https://github.com/CERTCC/SSVC/issues) explaining the information source and what decision point it informs.  
```

Comment:
Can we word this more carefully so it's more clear that we're asking for suggested categories of information sources, not specific feed products to subscribe to? I.e., we don't want this to turn into a solicitation for comments like "You left out Uncle Bob's Threat Feed which sources all the bad stuff from the darkest part of the dark web".

Reply:
This seems to be fixed in 51559c0ecfa7cd5f9d701e6a98a0c20a7bb5fc05